package/src/components/Chip/Chip.js

```diff
@@ -5,10 +5,10 @@ import { Chip as MuiChip, makeStyles } from "@material-ui/core";
 const useStyles = makeStyles((theme) => ({
   // Our chips are a little smaller, overriding medium and small sizes
   root: {
-    height: "30px"
+    height: 30
   },
   sizeSmall: {
-    height: "28px"
+    height: 28
```

Comment:
The Catalyst Chip has now been merged into master, so you can use the Catalyst one. After that, I believe these overrides shouldn't be necessary.

Reply:
Or should these defaults be edited in https://github.com/reactioncommerce/catalyst/blob/master/package/src/theme/defaultTheme.js#L294??

Reply:
I believe, these will still be necessary, as by default MUI's Chip is 30px(small) and 32(medium)

Reply:
I did not see those defaults, yes I'll edit those settings in the theme.